ecmascript/src/lib.rs

```diff
@@ -1,5 +1,5 @@
-pub use ast;
-pub use codegen;
-pub use parser;
-pub use preset_env;
-pub use transforms;
+pub use swc_ecma_ast;
+pub use swc_ecma_codegen;
+pub use swc_ecma_parser;
```

Comment:
Please change these to ast, codegen, parser, preset_env, trnasforms, like
`pub use swc_ecma_ast as ast;`